secure/storage/src/vault.rs

```diff
@@ -209,38 +128,12 @@ impl VaultStorage {
             .version)
     }
 
-    pub fn set_policies(
-        &self,
-        name: &str,
-        engine: &VaultEngine,
-        policy: &Policy,
-    ) -> Result<(), Error> {
-        for perm in &policy.permissions {
-            match &perm.id {
-                Identity::User(id) => self.set_policy(id, engine, name, &perm.capabilities)?,
-                Identity::Anyone => {
-                    self.set_policy(DIEM_DEFAULT, engine, name, &perm.capabilities)?
-                }
-                Identity::NoOne => (),
-            };
-        }
-        Ok(())
-    }
-
     fn crypto_name(&self, name: &str) -> String {
-        self.name(name, &VaultEngine::Transit)
+        name.replace('/', "__")
     }
 
-    fn secret_name(&self, name: &str) -> String {
-        self.name(name, &VaultEngine::KVSecrets)
-    }
-
-    fn name(&self, name: &str, engine: &VaultEngine) -> String {
-        if let Some(namespace) = &self.namespace {
-            format!("{}{}{}", namespace, engine.ns_seperator(), name)
-        } else {
-            name.into()
-        }
+    fn unnamespaced<'a>(&self, name: &'a str) -> &'a str {
```

Comment:
yeah, stuff like this is a bit scary

Reply:
I don't particularly think is scary, Its just how the vault storage was built that for some reason (that i'm not sure of because i'm not super familiar with vault) require that vault knows the unnamspaced key as well as the namespaced key.

Reply:
As long as this isn't ever exposed, we should be fine, no?